clusters/map.go

```diff
@@ -0,0 +1,47 @@
+package clusters
+
+import (
+	"github.com/deis/k8s-claimer/gke"
+	container "google.golang.org/api/container/v1"
+)
+
+// Map is a map from cluster name to GKE cluster
+type Map struct {
+	nameMap map[string]*container.Cluster
+}
+
+func clustersToMap(c []*container.Cluster) map[string]*container.Cluster {
+	ret := make(map[string]*container.Cluster)
+	for _, cluster := range c {
+		ret[cluster.Name] = cluster
+	}
+	return ret
+}
+
+// ParseMapFromGKE calls the GKE API to get a list of clusters, then returns a map representation
```

Comment:
in the description, just to be clear, say instead "... then returns a Map representation" (the capital "M" helps to distinguish it from a built-in map)